### PR TITLE
adding expfit stat and dictionarizing choice of statistic

### PR DESCRIPTION
**Pull request by tdent**

### CODE REVIEW

File: pycbc/events/stat.py [hunk 1]
```diff
@@ -28,33 +28,6 @@
 import numpy
 from . import events
 
-def get_statistic(option, files):
```
Comment (ahnitz) [inline pycbc/events/stat.py]: If you'd like a dictionary, please this method implemented but have it use the dictionary internally. This method is used by other codes than your own, and I'd prefer not to break compatibility.

Comment (tdent) [inline pycbc/events/stat.py]: What do you mean by 'other codes than my own'?  I don't own any codes, and git grep get_statistic in the pycbc repository gives me nothing. 
Also I don't see why you would want to define a new function merely to look up which class corresponds to a string.  That's what a dictionary does with 1/3 the number of lines of code, plus it allows access to a list of possible values which the function does not, plus it is simpler.

Comment (ahnitz) [inline pycbc/events/stat.py]: Tom, 

What I'm saying is simply to redefine this function to be what you've added to pycbc_coinc_findtrigs. This leaves the signature the same, and also avoids code duplication down the line. (i.e. the entire try except section)

Comment (tdent) [inline pycbc/events/stat.py]: OK, so the point of defining a function would be to ensure a more user-friendly error message in the possible future case of some other script wanting to use the same classes? 
At the moment there *is* nothing else to be compatible with.  In any case function calls should be able to change between code versions.

**Comment by tdent** [comment]
Have reinstated a 'get_statistic' function, although it returns a class rather than an instance.

**Pull request merged**